clusters/minikube/configure.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+SYSTEM_INSTALL_FLAGS="${SYSTEM_INSTALL_FLAGS:---node-port}"
+
+wait_for_ingress_ready() {
+  name=$1
+  namespace=$2
+
+  # nothing to do
+}
+
+# Install minikube cli
+source `dirname "${BASH_SOURCE[0]}"`/../../install.sh minikube
+
+export CHANGE_MINIKUBE_NONE_USER=true
```

Comment:
Maybe add a comment to explain what this is for?

Reply:
TBH, I don't know. The Minikube config is kribbed from https://github.com/LiliC/travis-minikube/blob/minikube-30-kube-1.12/.travis.yml

We used to have a comment with that link that was lost, I'll put it back.